modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1136,6 +1137,27 @@ def obj.class
           end
         end
       end
+
+      context 'when consumer is Veteran, but is missing a participant id' do
+        let(:parsed_codes) do
+          {
+            birls_id: '111985523',
+            participant_id: ''
+          }
+        end
+
+        it 'raises a 422, with message' do
+          with_okta_user(scopes) do |auth_header|
+            VCR.use_cassette('evss/claims/claims') do
+              VCR.use_cassette('evss/reference_data/countries') do
+                post path, params: data, headers: auth_header
+
+                expect(response.status).to eq(422)
```

Comment:
I'd also expect the message itself since we have a handful of 422's in that method (make sure it's the right 422).